src/Fabulous.XamarinForms/Views/Layouts/Frame.fs

```diff
@@ -0,0 +1,51 @@
+namespace Fabulous.XamarinForms
+
+open System.Runtime.CompilerServices
+open Fabulous
+open Fabulous.XamarinForms
+open Xamarin.Forms
+
+type IFrame =
+    inherit ITemplatedView
+
+module Frame =
+    let WidgetKey = Widgets.register<Frame> ()
+
+    let BorderColor =
+        Attributes.defineAppThemeBindable<Color> Frame.BorderColorProperty
+
+    let CornerRadius =
+        Attributes.defineBindable<float> Frame.CornerRadiusProperty
+
+    let HasShadow =
+        Attributes.defineBindable<bool> Frame.HasShadowProperty
+
+    let Content =
+        Attributes.defineBindableWidget Frame.ContentProperty
```

Comment:
This means you don't need this one. `Fabulous.Xamarin.Forms.ContentView.Content` will be used

Reply:
Done